jupyterhub/templates/proxy/secret.yaml

```diff
@@ -1,9 +1,9 @@
-{{- $manualHTTPS := (and .Values.proxy.https.enabled (eq .Values.proxy.https.type "manual")) }}
+{{- $manualHTTPS := and .Values.proxy.https.enabled (eq .Values.proxy.https.type "manual") -}}
 {{- if $manualHTTPS -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: manual-tls-proxy-{{ .Release.Name }}
+  name: proxy-manual-tls
```

Comment:
A convention for tls secrets is to use `example-com-tls` - hence the name change